packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -135,42 +125,36 @@ export class H5GroveApi extends DataProviderApi {
       return undefined;
     }
 
-    const { baseURL, params } = this.client.defaults;
-
-    const searchParams = new URLSearchParams(params as Record<string, string>);
-    searchParams.set('path', dataset.path);
-    searchParams.set('format', format);
-
-    if (selection) {
-      searchParams.set('selection', selection);
-    }
+    const searchParams = new URLSearchParams({
+      ...(this.axiosConfig?.params as Record<string, string>),
```

Comment:
Sorry, had to push a small linting fix here. Not ideal, but I'll deal with this export feature later as explained.